.github/workflows/tag_image_push.yml

```diff
@@ -28,37 +30,52 @@ jobs:
         with:
           version: v0.5.1
 
-
       - name: Login to Docker Hub
         run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
 
       - name: Build/Push OSS image
         run: |
-          docker buildx build --push \
-            --tag hazelcast/hazelcast:${{ env.RELEASE_VERSION }} \
-            --platform=linux/arm64,linux/amd64,linux/ppc64le,linux/s390x hazelcast-oss
+          if [[ "${{ env.PUSH_LATEST }}" == "yes" ]]; then
+            docker buildx build --push \
+              --build-arg HZ_VERSION=${{env.RELEASE_VERSION }} \
+              --tag hazelcast/hazelcast:${{ env.RELEASE_VERSION }} \
+              --tag hazelcast/hazelcast:latest \
+              --platform=linux/arm64,linux/amd64,linux/ppc64le,linux/s390x hazelcast-oss
+          else
+            docker buildx build --push \
```

Comment:
So this push does not change the `latest` tag, am I correct?

Reply:
Yes, it doesn't affect the `latest` tag.